content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Mosaic visualization
+list_title: Mosaic
+list_image: /img/influxdb/2-0-visualizations-mosaic-example.png
+description: >
+  The Mosaic view displays the single value most recent value for a time series in a mosaic view.
+weight: 206
+menu:
+  influxdb_2_0:
+    name: Mosaic
+    parent: Visualization types
+---
+
+The **Mosaic** visualization displays data from two or more qualitative variables.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-mosaic-example.png" alt="mosaic example" />}}
+
+Select the **Mosaic** option from the visualization dropdown in the upper left.
+
+## Mosaic behavior
+The mosaic visualization displays string values in a graph.
+
+
+## Mosaic controls
+To view **Mosaic** controls, click **{{< icon "gear" >}} Customize** next to the visualization dropdown.
+
+###### Data
+- **Fill Column**: Select a column to fill in the mosaic tiles.
+- **X Column**: Select a column to display on the x-axis.
+- **Y Column**: Select one or more columns to display on the y-axis.
+- **Time Format**: Select the time format. Options include:
```

Comment:
@TCL735 is this now updated with many more? maybe we just link to a page with all the formats on them?

Reply:
Yes. Time Format now has many more options. We can address this in a separate PR that updates this on all pages for all graphs types.

Reply:
@noramullen1 We could create a shortcode for the list of time values so we only have to edit it in one place. Doesn't have to happen in this PR, but it'd be good to make this a little more maintainable.

Reply:
Sounds good, created an issue: https://github.com/influxdata/docs-v2/issues/2249